File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n = 0;
    do
    {
        n = get_int("Enter the number of bricks : ");
    }
    while(n <= 0 || n > 8);

    for (int i = 0; i < n; i++)
    {
        int num = 0;
        for (int j = 0; j < 2*n+2; j++)
        {
            if ( (j < n && j >= n-i-1) || (j >= n + 2 && j <= n+2+i))
            {
                printf("#");
                num++;
            }
            else if(num == 2*(i+1))
            {
                break;
            }
            else
            {
                printf(" ");
            }
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

while True:
    n = get_int("Enter the number of rows: ")
    if n >= 1 and n <= 8:
        break

for i in range(n):
    num = 0
    for j in range(2*n + 2):
        if (j < n and j >= n - i - 1) or (j >= n + 2 and j <= n + 2 +i):
            print("#", end="")
            num += 1
        elif num == 2*(i+1):
            break
        else:
            print(" ", end="")
    print()